tests/integration/models_utils.py

```diff
@@ -49,6 +49,7 @@ class Meta:
     server_domain: str = factory.LazyFunction(lambda: settings.OFF_SERVER_DOMAIN)
     server_type = "off"
     unique_scans_n = 10
+    annotation = 0
```

Comment:
No annotation should be None by default… annotation = 0 means it has been annotated as "false". And we don't want such a default.

If you need it locally, use it as a parameter `ProductInsightFactory(annotation=0)`